pages/blog/socketio-automatic-docs.md

```diff
@@ -0,0 +1,553 @@
+---
```

Comment:
please add 2 missing props:
- `featured: true`
- `date` as we need to specify when it is published. Please set next Thursday